Services/UI/OJS.Services.Ui.Business/Implementations/SubmissionsBusinessService.cs

```diff
@@ -441,47 +455,103 @@ public async Task ProcessExecutionResult(SubmissionExecutionResult submissionExe
         submission.Processed = true;
         submission.ProcessingComment = null;
 
+        using var scope = new TransactionScope(TransactionScopeAsyncFlowOption.Enabled);
         if (executionResult != null)
         {
             ProcessTestsExecutionResult(submission, executionResult);
+
+            await this.submissionsForProcessingData.MarkProcessed(submission.Id);
```

Comment:
The submission should be marked as processed, even when there is `submissionExecutionResult.Exception`

Reply:
Done